infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesClientConfigFactory.java

```diff
@@ -9,24 +9,22 @@
  * Contributors:
  *   Red Hat, Inc. - initial API and implementation
  */
-package org.eclipse.che.workspace.infrastructure.openshift;
+package org.eclipse.che.workspace.infrastructure.kubernetes;
```

Comment:
maven module openshift and package k8s. Can you explain your motivation?

Reply:
the file is moved to kubernetes module. It has become more generic and will be used for both k8s and openshift.